cmd/app.go

```diff
@@ -8,8 +8,8 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 
-	"github.com/rebuy-de/kubernetes-deployment/kubernetes"
-	"github.com/rebuy-de/kubernetes-deployment/settings"
+	"github.com/rebuy-de/kubernetes-deployment/pkg/kubernetes"
+	"github.com/rebuy-de/kubernetes-deployment/pkg/settings"
```

Comment:
Because in this way it is easier to distinguish between golang packages and other directories, like `hack` and `target`.